fastapi_views/models.py

```diff
@@ -58,93 +65,80 @@ def new(cls: type[Self], detail: str, **kwargs: Any) -> Self:
     type: Union[Url, Literal["about:blank"]] = Field(
         "about:blank", description="Error type"
     )
-    title: Optional[str] = Field("Bad Request", description="Error title")
+    title: str = Field(None, description="Error title")
     status: int = Field(HTTP_400_BAD_REQUEST, description="Error status")
-    detail: str = Field(description="Error detail")
+    detail: str = Field(None, description="Error detail")
```

Comment:
Wiem że pydatic default nie sprawdza, ale gryzie mi się to troche z typowanie mimo wszystko chyba powinno zostać Optional

Reply:
No mi się żadna opcja nie podoba, bo te pola są uzupełniane przez model validator i teraz jak to zrobić żeby schema w openapi nie była optional? :)

Reply:
To dlaczego ustawiasz default na None ?

Reply:
A jaka jest inna opcja? Ustawic na `""` ?

Reply:
title: Bad request 
detail: Something wrong
Skoro na status masz default sensowny to i dla reszty może być